src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" autocomplete="off">
+      <fieldset class="field personal-info">
+        <legend>Personal information</legend>
+        <div class="first form-field">
+          <label for="user-surname">Surname:</label>
+          <input type="text" name="surname" id="user-surname">
+        </div>
+        <div class="form-field">
+          <label for="user-name">Name:</label>
+          <input type="text" name="name" id="user-name">
+        </div>
+        <div class="form-field">
+          <label for="user-age">How old are You?</label>
+          <input 
+          type="number" 
+          name="age" 
+          id="user-age" 
+          min="1" 
+          max="100" 
+          value="12"
+       ></div>
```

Comment:
```suggestion
          <input 
            type="number" 
            name="age" 
            id="user-age" 
            min="1" 
            max="100" 
            value="12"
          >
        </div>
```
Форматування. 
Почитати можна тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation

Reply:
@varseniuk 

1. Властивості об'єктів позношу зараз 2 на од.
Але ще питання:
Чи правильно я зрозумів, що закриваючий тег дів треба знести на окрему строку ? 
У відео Міша робив саме так:
></div>